Setup works just fine with the version of setuptools_scm in Void.

--- a/setup.py
+++ b/setup.py
@@ -111,7 +111,7 @@
     # for example:
     # $ pip install -e .[dev,test]
     extras_require={'doc': read_requirements('doc')},
-    setup_requires=['setuptools_scm<6.0'],
+    setup_requires=['setuptools_scm'],
     use_scm_version=True,
 
     # If there are data files included in your packages that need to be
